bc3_importer/data/product_data.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+    <record id="product_product_product_template_units" model="product.template">
```

Comment:
If this product are wildcards when no other matches is found, please put them as inactive for not having them in the product list, and only added programmatically.

Reply:
Although the products are generic, I have not put them as inactive as some companies wanted to modify them to suit their business (e.g. invoicing policy).

Reply:
Well, that can be done applying the inactive filter, but it's not blocking.